setup.py

```diff
@@ -38,7 +38,7 @@
         "sympy",
         # pinned for compatibility with strawberry fields
         "antlr4-python3-runtime==4.9.2",
-        "amazon-braket-schemas>=1.21.0",
+        "amazon-braket-schemas>=1.21.2",
```

Comment:
I believe the [schemas](https://github.com/amazon-braket/amazon-braket-schemas-python) latest version is 1.21.3 if you want to pull in the latest here

Reply:
This is pinned for the version where local detuning rename happened